prowler/providers/aws/services/s3/s3_bucket_cross_region_replication/s3_bucket_cross_region_replication.py

```diff
@@ -0,0 +1,39 @@
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.s3.s3_client import s3_client
+
+
+class s3_bucket_cross_region_replication(Check):
+    def execute(self):
+        findings = []
+        for arn, bucket in s3_client.buckets.items():
+            report = Check_Report_AWS(self.metadata())
+            report.region = bucket.region
+            report.resource_id = bucket.name
+            report.resource_arn = arn
+            report.resource_tags = bucket.tags
+
+            if (
+                bucket.versioning
+                and bucket.replication
+                and bucket.replication.status == "Enabled"
+                and bucket.replication.destination
+            ):
+                destination_bucket = s3_client.buckets.get(
```

Comment:
What if the bucket is in another account or if Prowler is only auditing one region?

Reply:
Solved with my commit.